main.go

```diff
@@ -88,7 +88,7 @@ func init() {
 
 	if config.Slack.WebhookURL != "" {
 		var err error
-		slackClient, err = outputs.NewClient("Slack", config.Slack.WebhookURL, config, stats, promStats, statsdClient, dogstatsdClient)
+		slackClient, err = outputs.NewClient("Slack", config.Slack.WebhookURL, false, config, stats, promStats, statsdClient, dogstatsdClient)
```

Comment:
```suggestion
		slackClient, err = outputs.NewClient("Slack", config.Slack.WebhookURL, config.Slack.MutualTls, config, stats, promStats, statsdClient, dogstatsdClient)
```

You should use this kind of pattern.

BTW, you didn't changed the `SlackOutputConfig` struct neither `config.go` for